Pod/Classes/BONTag.h

```diff
@@ -0,0 +1,69 @@
+//
+//  BONTag.h
+//  Pods
+//
+//  Created by Nora Trapp on 5/11/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import "BONCompatibility.h"
+
+@protocol BONTextable;
+
+@interface BONTag : NSObject <NSCopying>
+
+@property (copy, nonatomic, readonly, BONNonnull) NSString *startTag;
+@property (copy, nonatomic, readonly, BONNonnull) NSString *endTag;
+@property (copy, nonatomic, readonly, BONNonnull) NSString *escapeString;
+@property (copy, nonatomic, readonly, BONNonnull) id<BONTextable> textable;
+
+- (BONNonnull instancetype)initWithStartTag:(BONNonnull NSString *)startTag endTag:(BONNonnull NSString *)endTag escapeString:(BONNonnull NSString *)escapeString textable:(BONNonnull id<BONTextable>)textable;
```

Comment:
The other thing I was thinking of was apparently `NS_REFINED_FOR_SWIFT`, but to use it you still have to provide a Swift extension yourself - there's no magic that happens on import from the Obj-C header.